src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,110 @@
+package seedu.addressbook.ui;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Scanner;
+
+import static seedu.addressbook.common.Messages.*;
+
+/**
+ * Formats text for display.
+ */
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /**
+     * Returns formatted version of the messages.
+     */
+    public String format(String... message){
```

Comment:
Ideally, every public method should have a header comment, along the `@param` `@return` when available